Changelog v1.8.x.md

```diff
@@ -11,6 +12,10 @@
 - Fix: Incorrect version check with VersionChecker mod integration
 - Fix: Cage Trap owner collision is incorrect
```

Comment:
Your cage trap changes are not reflected in the changelog, so I guess you can just remove "owner" from this line.